### PR TITLE
[RB-16] About us page

### PR DESCRIPTION
**Pull request by liuba83**
## Description
Create About us page

## Changes
Create About.jsx component

## Screenshots
Screen Shot 2024-11-20 at 10 41 52 PM

### CODE REVIEW

File: src/assets/images/square.png
Comment (in43sh) [inline src/assets/images/square.png:1]: I understand that this file was here for development purposes but we can't merge such file into the `dev` branch. You would have to replace it with a default photo or something else.

File: src/components/TeamMemberCard.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+// import avatar from '../assets/images/default-avatar.jpeg';
```
Comment (in43sh) [inline src/components/TeamMemberCard.jsx]: You should always remove comments like this before creating a new PR.

File: src/components/TeamMemberCard.jsx
Comment (in43sh) [inline src/components/TeamMemberCard.jsx:1]: You should run command `npm run format` as some css classes are not in order.

File: src/pages/About.jsx
Comment (in43sh) [inline src/pages/About.jsx:1]: Same here, some css classes are not in order. `npm run format` will fix it.

File: src/pages/About.jsx [hunk 2]
```diff
@@ -1,7 +1,113 @@
 import React from 'react';
```
Comment (in43sh) [inline src/pages/About.jsx:1]: You can also delete this line

File: src/components/TeamMemberCard.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```
Comment (in43sh) [inline src/components/TeamMemberCard.jsx]: You can delete this line as beginning from recent versions you don't need to import React in your component.

File: src/components/TeamMemberCard.jsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+// import avatar from '../assets/images/default-avatar.jpeg';
+// import avatar from '../assets/images/square.png';
+
+const TeamMemberCard = ({ img, name, role }) => {
+  return (
+    <div className="max-w-sm rounded overflow-hidden shadow-lg mb-10 bg-lightBlue py-5 px-6">
```
Comment (in43sh) [inline src/components/TeamMemberCard.jsx]: I think you can remove these two tailwind classes: `max-w-sm overflow-hidden`.

File: src/pages/About.jsx [hunk 5]
```diff
@@ -1,7 +1,113 @@
...
+      role: 'Mentor',
+    },
+    {
+      image: avatar,
+      name: 'Vadim',
+      role: 'Mentor',
+    },
+  ];
+
+  const mentorCards = mentors.map((item, index) => (
```
Comment (in43sh) [inline src/pages/About.jsx]: I think this should be called `mentorsCards`

File: src/pages/About.jsx [hunk 6]
```diff
@@ -1,7 +1,113 @@
...
+        Meet our team
+      </h2>
+      <div className="flex flex-col md:flex-row justify-between gap-2 md:gap-16">
+        {frontendCards}
+      </div>
+      <div className="flex flex-col md:flex-row justify-around gap-2 md:gap-32">
+        {backendCards}
+      </div>
+      <div className="flex flex-col md:flex-row justify-between gap-2 md:gap-16">
+        {mentorCards}
```
Comment (in43sh) [inline src/pages/About.jsx]: Replace it with `mentorsCards`.

File: src/pages/About.jsx [hunk 7]
```diff
@@ -1,7 +1,113 @@
 import React from 'react';
+import TeamMemberCard from '../components/TeamMemberCard';
+import avatar from '../assets/images/square.png';
 
 const About = () => {
-  return <></>;
+  const frontend = [
+    {
+      image: avatar,
+      name: 'Evgenii',
```
Comment (in43sh) [inline src/pages/About.jsx]: You should add people's last names here.

File: src/pages/About.jsx [hunk 8]
```diff
@@ -1,7 +1,113 @@
...
+    <TeamMemberCard
+      key={index}
+      img={item.image}
+      name={item.name}
+      role={item.role}
+    />
+  ));
+
+  return (
+    <div className="w-full flex flex-col items-center bg-white">
```
Comment (in43sh) [inline src/pages/About.jsx]: You don't need `bg-white` as the page is already white by default.

File: src/pages/About.jsx [hunk 9]
```diff
@@ -1,7 +1,113 @@
...
+      <div className="flex flex-col md:flex-row justify-around gap-2 md:gap-32">
+        {backendCards}
+      </div>
+      <div className="flex flex-col md:flex-row justify-between gap-2 md:gap-16">
+        {mentorCards}
+      </div>
+      <h2 className="font-headings font-bold text-3xl md:text-4xl my-7 md:my-10">
+        About Re:Books
+      </h2>
+      <p className="px-10 pb-10 flex flex-col items-center w-full font-body text-justify">
```
Comment (in43sh) [inline src/pages/About.jsx]: Minor Tailwind suggestions:
- Horizontal padding should be 60 pixels on the desktop version
- `flex w-full flex-col items-center` can be removed
- Font size should be 24px and line height should be 52px

File: src/pages/About.jsx [hunk 10]
```diff
@@ -1,7 +1,113 @@
 import React from 'react';
+import TeamMemberCard from '../components/TeamMemberCard';
+import avatar from '../assets/images/square.png';
 
 const About = () => {
-  return <></>;
+  const frontend = [
```
Comment (in43sh) [inline src/pages/About.jsx:13]: There could be a better way to do it, but this will work for now

**Review by in43sh — CHANGES_REQUESTED** [review]
Good work overall! Some minor changes needed but still you did a good job!

File: src/pages/About.jsx (hunk 2 above)
Comment (in43sh) [reply src/pages/About.jsx:1]: Evegenii and I talked offline and we don't need to remove it anymore.

**Review by in43sh — APPROVED** [review]

**Review by VadimDmitr — APPROVED** [review]
Great job, Liuba! If yesterday I had questions about the font, lineHeight, and text at the bottom of the page, today I can see that everything is well-adjusted and looks excellent across all devices.

**Review by in43sh — APPROVED** [review]

**Review by VadimDmitr — APPROVED** [review]
Awesome!

**Pull request merged**